src/vma/lwip/tcp_out.c

```diff
@@ -1343,8 +1343,13 @@ tcp_rst(u32_t seqno, u32_t ackno, u16_t local_port, u16_t remote_port, struct tc
   tcphdr->dest = htons(remote_port);
   tcphdr->seqno = htonl(seqno);
   tcphdr->ackno = htonl(ackno);
-  TCPH_HDRLEN_FLAGS_SET(tcphdr, TCP_HLEN/4, TCP_RST | TCP_ACK);
-  tcphdr->wnd = PP_HTONS(( TCP_WND  & 0xFFFF ));
+  if (!flags) {
+      tcphdr->wnd = PP_HTONS(( TCP_WND  & 0xFFFF ));
+      TCPH_HDRLEN_FLAGS_SET(tcphdr, TCP_HLEN/4, TCP_RST | TCP_ACK);
+  } else {
+      tcphdr->wnd = 0;
+      TCPH_HDRLEN_FLAGS_SET(tcphdr, TCP_HLEN/4, *flags);
+  }
```

Comment:
tcphdr->wnd = 0; is this unique behavior for this case ? or all reset should be sent we wnd = 0

Reply:
The behavior was aligned with Linux (i.e. wnd=0) in LISTEN state < S. (SYN-ACK)